pafm.php

```diff
@@ -236,14 +273,15 @@ function escape($uri){
 }
 function removeQuotes($subject, $single = true, $double = true) {
 	if ($single)
-		$subject = str_replace('\'', null, $subject);
+		$subject = str_replace('\'', '', $subject);
 	if ($double)
-		$subject = str_replace('"', null, $subject);
+		$subject = str_replace('"', '', $subject);
 	return $subject;
 }
 function return_bytes($val) { //for upload. http://php.net/ini_get
     $val = trim($val);
-    $last = strtolower($val{strlen($val)-1});
+    $last = strtolower($val[strlen($val)-1]);
+    $val = (int) $val;
```

Comment:
php 8 compliance, because `string` type needed to be converted to number, before making any arithmetic operations